code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -4895,6 +4895,12 @@
 	nutriment_factor = FOOD_METABOLISM
 	color = "#664300" //rgb: 102, 67, 0
 
+/datum/reagent/drink/silencer/on_mob_life(var/mob/living/M)
+
+ 	if(..())
+            	 return 1
```

Comment:
The problem is indentation on this line and the next one, but I'm not actually sure what's wrong with it. Might you have indented with spaces? That's what it looks like on this page but I'm not sure if that's just a GitHub thing or what.